libcontainer/rootfs_linux.go

```diff
@@ -182,6 +182,33 @@ func mountCmd(cmd configs.Command) error {
 	return nil
 }
 
+func prepareBindMounts(m *configs.Mount, rootfs string) error {
```

Comment:
I think it should be prepareBindMount without s, because it handles only one mount

Reply:
done